frontend/src/views/Datasets/DatasetData.js

```diff
@@ -3,15 +3,18 @@ import React from 'react';
 import { Box } from '@mui/material';
 import DatasetTables from './DatasetTables';
 import DatasetFolders from './DatasetFolders';
+import config from '../../generated/config.json';
 
 const DatasetData = ({ dataset, isAdmin }) => (
   <Box>
     <Box>
       <DatasetTables dataset={dataset} isAdmin={isAdmin} />
     </Box>
-    <Box sx={{ mt: 3 }}>
-      <DatasetFolders dataset={dataset} isAdmin={isAdmin} />
-    </Box>
+    {config.modules.datasets.features.file_actions === true && (
```

Comment:
Can you add null check here?
It may cause errors if the configs file isn't defined properly

Reply:
I believe we should create a utility function called `bool isModuleEnabled(moduleName){}` that does this logic, instead of repeating the almost the same code everywhere (DRY). With a switch statement for module name, and it's the only one which will import the `configs.json` file. What do you think?

Reply:
that makes sense to me, any modification we want to make in the logic can then be restricted to the new utility function. Will update the PR.